### PR TITLE
Remove feature flag for the new add-a-company journey

### PR DESCRIPTION
**Pull request by alixedi**
### Description of change

This removes the feature flag that enabled the new add-a-company journey.

The new add-a-company journey is now the default way to add a new company in Data Hub. The old journey for creating new companies in Data Hub have already been removed.

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.md](https://github.com/uktrade/data-hub-api/blob/master/changelog/README.md) for instructions
* [ ] Do any added or updated endpoints appear in the API documentation? See [docs/Maintaining the API documentation.md](https://github.com/uktrade/data-hub-api/blob/develop/docs/Maintaining&#32;the&#32;API&#32;documentation.md) for more details
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: changelog/company/add-a-company-feature-flag.removal.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This removed the feature flag that enabled the new add-a-company journey.
+
+The new add-a-company journey is now the default way to add a new company in Data Hub. The old journey for creating new companies in Data Hub have already been removed.
```
Comment (elcct) [inline changelog/company/add-a-company-feature-flag.removal.md]: This should say `The old journey for creating new companies in Data Hub has already been removed.`?

File: changelog/company/add-a-company-feature-flag.removal.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This removed the feature flag that enabled the new add-a-company journey.
```
Comment (reupen) [inline changelog/company/add-a-company-feature-flag.removal.md]: Don't think the wording and tense here works well for a news fragment – would reword it to something like 'The feature flag that...was removed.'
